redaxo/src/core/lang/de_de.lang

```diff
@@ -291,26 +239,10 @@ setup_password_hint = Zur Vergabe eines neuen Passwortes dieses eintragen. Sonst
 hsts_more_information = HSTS sollte in den Hostingeinstellungen der Domain/s (empfohlen) aktiviert werden, kann aber auch in der config.yml von REDAXO nachträglich aktiviert werden. Weitere Informationen in der REDAXO-Doku.
 
 # redaxo\include\pages\addon.php
-addon = AddOn
 addons = AddOns
 package_help = Hilfe für
 package_no_help_file = Keine Hilfedatei gefunden
 package_back = zurück
-addon_no_install = AddOn <b>{0}</b> konnte aus folgendem Grund <b>nicht</b> installiert werden:
```

Comment:
https://github.com/redaxo/redaxo/blob/02fd86dfb681bc541fb214d6a8c8f00f17fcd19f/redaxo/src/core/lib/packages/manager.php#L190

`$this->i18n` im package-manager kann immer `addon_*` oder `plugin_*` sein.

Reply:
Ich löse es erst einmal so: Falls der Key nicht gefunden wird, prüfe ob dieser mit Prefix `addon_` oder `plugin_` oder `package_` vorhanden ist. Wenn ja bleibt dieser erhalten. Unabhängig ob er via `$this->` oder `::msg` in den Dateien verwendet wird.